### ISSUE TITLE
not working because motion data

### ISSUE THREAD

**Issue by microbians** [OrthoCameraApp/OrthoCameraApp.github.io#1]
not working because motion data is not available or I  not be able to grant ... have not the option... i have iphone 6s

**Comment by weihengpan** [comment]
It may happen if you pressed the home button when the app is asking for permissions during first launch. In this case, you may use the following method:

1. Uninstall Ortho.
2. Go to Settings > General > Reset > Reset Location and Privacy Settings. (Note this will reset privacy settings of ALL your apps. Unfortunately you cannot just reset one of them.)
3. Reinstall Ortho, launch it and give the appropriate permissions.

Hope this helps.

**Comment by microbians** [comment]
Yes sorry not post it before, I have some system location services off and so activity was not shown in the app settings to approve 😉 

solved, thanks!

Gabriel Suchowolski
http://domestika.org/microbians
